### PR TITLE
minor spelling tweaks

### PR DESCRIPTION
**Pull request by brettkoonce**

### CODE REVIEW

**Review by compnerd — APPROVED** [review]

File: docs/ByteTree.md [hunk 1]
```diff
@@ -13,20 +13,20 @@ For example, the string "Hello World" would be encoded as `(uint32_t)11` `"Hello
...
 ## Serialization of objects
 
-An object consists of its size, measured in the number of fields and represented as a `uint_32t` in little endian order, followed by the direct concatenation of its fields. Because each field is again prefixed with its size, no delimites are necessary in between the fields.
+An object consists of its size, measured in the number of fields and represented as a `uint_32t` in little endian order, followed by the direct concatenation of its fields. Because each field is again prefixed with its size, no delimits are necessary in between the fields.
 
-To distinguish scalars and objects, the size of objects has its most-siginificant bit set to 1. It must be ignored to retrieve the number of fields in the object.
+To distinguish scalars and objects, the size of objects has its most-significant bit set to 1. It must be ignored to retrieve the number of fields in the object.
 
-Arrays are modelled as objects whose fields are all of the same type and whose length is variadic (and is indicated by the object's size).
+Arrays are modeled as objects whose fields are all of the same type and whose length is variadic (and is indicated by the object's size).
```
[the remark this answers is not in the document]
Comment (brettkoonce) [reply docs/ByteTree.md]: reverted

File: docs/ABI/TypeMetadata.rst [hunk 2]
```diff
@@ -137,7 +137,7 @@ contain the following fields:
 - The optional **parameter flags vector** begins after the end of
   **parameter type vector** and consists of **NumParameters** 32-bit flags.
   This includes information such as whether the parameter is ``inout`` or
-  whether it is variadic.  The presence of this vector is signalled by a flag
+  whether it is variadic.  The presence of this vector is signaled by a flag
```
[the remark this answers is not in the document]
Comment (brettkoonce) [reply docs/ABI/TypeMetadata.rst]: reverted

**Comment by compnerd** [comment]
@swift-ci please smoke test

**Comment by compnerd** [comment]
@swift-ci please smoke test

**Pull request merged**